### PR TITLE
Week 4 HW

### PR DESCRIPTION
**Pull request by kamibena**
Week four complete!

### CODE REVIEW

**Comment by kamibena** [comment]
I have the pdf created for week four and ready to make a new pull request whenever needed.   Does not seem I can make another while my previous request is still open?

**Comment by klocey** [comment]
@kamibena I'm going to merge this pull request and make a few comments and helpful suggestions on your work. You won't need to revise/resubmit unless I explicitly ask. Thanks!

File: Week4-Spatial/spatial_assignment.Rmd [hunk 1]
```diff
@@ -102,76 +121,122 @@ In the R code chunk below, do the following:
...
 
 # 6. Plot the refuge pond locations
-
+plot(Refuge.Ponds,line='r',col='red',pch=20,cex=1.5,add=T)
 ```
 
 ***Question 4a***: What are datums and projections?
 
-> ***Answer 4a***:  
+> ***Answer 4a***:  A datum is basically a point of refernce on the globe that the projection uses to plot the reference point on a 2D map.
```
Comment (klocey) [inline Week4-Spatial/spatial_assignment.Rmd]: @kamibena Not really. A datum is a model for the shape of the Earth. Over time, datum's are refined to better reflect Earth's shape and to achieve better accuracy for particular regions. This is the case for the North American datums (NAD 1927, NAD 1983), which is commonly used by GPS devices.

Projections are ways of representing a 3D shape (e.g., Earth) or a model of a 3D shape (e.g., NAD83) on a 2D surface (i.e., map).

File: Week4-Spatial/spatial_assignment.Rmd [hunk 2]
```diff
@@ -102,76 +121,122 @@ In the R code chunk below, do the following:
...
-> ***Answer 5***:
+> ***Answer 5***:  The internal correlation of a variable through space.  In other words, it is a measure of how much a viable correlates with itself over space.
 
 
 ## 6) EXAMINING DISTANCE-DECAY
 
 ***Question 6***: In your own words, explain what a distance decay pattern is and what it reveals.
 
-> ***Answer 6***:
+> ***Answer 6***:  The distance-decay pattern describes the diminishing interactions as two species/places/things of interest become more spatially distant.
```
Comment (klocey) [inline Week4-Spatial/spatial_assignment.Rmd]: @kamibena And a basic pattern of spatial autocorrelation. In fact, the distance decay pattern is what led folks to consider more formal statistics of autocorrelation.

File: Week4-Spatial/spatial_assignment.Rmd [hunk 3]
```diff
@@ -102,76 +121,122 @@ In the R code chunk below, do the following:
...
+
+# 10) Use `simba` to calculates the difference in slope or intercept of two regression lines
+diffslope(env, struc, dist, struc)
 ```
 
 
 ***Question 7***: What can you conclude about community similarity with regars to environmental distance and geographic distance?
 
-> ***Answer 7***: 
+> ***Answer 7***: There is a slight negative relationship between community similarity and environmental distance, but the community stay relatively stable over geographic distance.  Accoutning for both simultaneouly, there is a significant slope (p < 0.001) that shows for every unit of change, species composition decreases by 0.17%
```
Comment (klocey) [inline Week4-Spatial/spatial_assignment.Rmd]: @kamibena It's worth noting that the range of values on the x-axes differs greatly between environmental distance and geographic distance. In this case, one could transform the x-axis data (e.g., using log values, relativizing each value by the maximum value, etc.).

File: Week4-Spatial/spatial_assignment.Rmd [hunk 4]
```diff
@@ -102,76 +121,122 @@ In the R code chunk below, do the following:
...
 
 
 ## 7) EXAMINING SPECIES SPATIAL ABUNDANCE DISTRIBUTIONS
 
 ***Question 8***: In your own words, explain the species spatial abundance distribution and what it reveals.
 
-> ***Answer 8***:
+> ***Answer 8***: The species spatial abundance distrbution describes how individual organisms are arranged in a given landscape.  It can inform us about the rate and abundances at which you might encounter organisms in that particular space.
```
Comment (klocey) [inline Week4-Spatial/spatial_assignment.Rmd]: @kamibena Nice. Additionally, it's a spatially implicit pattern of aggregation of a population's abundance or density. Spatially implicit means that exact geographic locations aren't taken into account. SSADs generally reveal high abundances in relatively few location and low abundances (or absences) in many or most locations. Extent and grain are important to consider.

File: Week4-Spatial/spatial_assignment.Rmd [hunk 5]
```diff
@@ -196,26 +279,42 @@ Many patterns of biodiversity relate to spatial scale.
 
 ***Question 9***:  List, describe, and give examples of the two main aspects of spatial scale
 
-> ***Answer 9***: 
+> ***Answer 9***: Two main aspects: Extent, Grain.  Extent describes the greatest distance in a given study area. Grain, on the other hand, is the smallest distance in a given study area. For example if you are measuring biodiversity in a 20ha plot the extent is 20ha.  If you were measuring diversity per hectare in that 20ha plot, then 1ha would be the grain of that study.  As another example, if you were measuring diversity along 100km interstate and had that study site partitioned into 0.25km plots, then the extent is 100km and the grain is 0.25km.
```
Comment (klocey) [inline Week4-Spatial/spatial_assignment.Rmd]: 👍

File: Week4-Spatial/spatial_assignment.Rmd [hunk 6]
```diff
@@ -239,18 +385,32 @@ In the R code chunk below, provide the code to:
...
+z <- round(results[[1]][[2]],2)
+legend(x=2, y=2, paste(c('slope = ', z), collapse = " "), cex=0.8,
+       box.lty=0)
 ```
 
 
 ***Question 10a***: Describe how richness relates to area in our simulated data by interpreting the slope of the SAR.
 
-> ***Answer 10a***: 
+> ***Answer 10a***: There is a positive relationship between species richness and area, so as the area of the land increases species richness increases as well.
```
Comment (klocey) [inline Week4-Spatial/spatial_assignment.Rmd]: @kamibena Directions were to interpret the slope. If the slope is *z* = 0.22, then what does that mean? Remember, the general form of the SAR is *S* = *cA*^*z*. So, log-transforming both sides (as in the corresponding figure) leads to log(*S*) = log(*c*) + *z*log(*A*).

If area (A) increases by an order of magnitude (i.e., power of 10) then could we say that richness scales to nearly the fifth power (0.2) of area *based solely on random placement*? Could this give us some expectation against which to compare empirical patterns?

File: Week4-Spatial/spatial_assignment.Rmd [hunk 7]
```diff
@@ -239,18 +385,32 @@ In the R code chunk below, provide the code to:
...
 
 ***Question 10b***: What does the y-intercept of the SAR represent?
 
-> ***Answer 10b***: 
+> ***Answer 10b***: The Y-intercept represents the log transformed values of species richness in a given area of the dataset.
```
Comment (klocey) [inline Week4-Spatial/spatial_assignment.Rmd]: @kamibena y-axis: log-transformed species richness, i.e., log-transformed alpha diversity. Ideally, we would generated the SAR starting from different locations/sites/corners to obtain an average S for each scale.

Note, if using log10 scale, you would raise 10 to the value on the y-axis to obtain the value of S at that spatial scale (i.e., corresponding value of the x-axis).

**Pull request merged**